pkg/parse/run_test.go

```diff
@@ -427,41 +427,30 @@ func TestRun(t *testing.T) {
 }
 
 func TestBackoffRetryCount(t *testing.T) {
-	parser := newParser(t, FileSource{}, false, 10*time.Microsecond, 150*time.Microsecond)
+	parser := newParser(t, FileSource{}, false, 1*time.Nanosecond, 15*time.Nanosecond)
 	testState := &namespacecontroller.State{}
-	reimportCount := 0
 	retryCount := 0
-	testIsDone := func(reimportCount *int, _ *int) bool {
-		return *reimportCount == 35
-	}
 
 	t.Logf("start running test at %v", time.Now())
 
-	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
defer the cancel to make sure the timer is cleaned up at the end of the function, even if it panics, without waiting for the timeout.

Reply:
Done